server/database/createTable.sql

```diff
@@ -219,6 +219,7 @@ INSERT INTO task (completed, title)
 VALUES (false, 'make dinner');
 CREATE TABLE IF NOT EXISTS "view" (
     "id" SERIAL PRIMARY KEY,
+    "name" text,
```

Comment:
this adds the name column to the view table in the sql file for creating the database schema.